flow/connectors/postgres/cdc.go

```diff
@@ -361,6 +367,16 @@ func PullCdcRecords[Items model.Items](
 		return nil
 	}
 
+	var fetchedBytesCounter metric.Int64Counter
+	if p.otelManager != nil {
+		var err error
+		fetchedBytesCounter, err = p.otelManager.GetOrInitInt64Counter(otel_metrics.BuildMetricName(otel_metrics.FetchedBytesCounterName),
+			metric.WithUnit("By"), metric.WithDescription("Bytes received of CopyData over replication slot"))
+		if err != nil {
+			logger.Error("Could not get FetchedBytesCounter", slog.Any("error", err))
+		}
```

Comment:
we should also probably sent out an alert on this as it will be an important metric

Reply:
made it return instead of ignore